maven/bnd-baseline-maven-plugin/pom.xml

```diff
@@ -45,6 +45,14 @@
 			<groupId>biz.aQute.bnd</groupId>
 			<artifactId>biz.aQute.bndlib</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Why is this change needed? There are no other baseline plugin changes in this PR.

Reply:
This is because of the change in the transitive dependencies of the bnd-maven-plugin due to the addition of `bnd-shared-maven-lib`. The dependencies are needed simply to satisfy the maven resolver.

Reply:
So because bnd-maven-plugin use bnd-shared-maven-lib, bnd-baseline-maven-plugin must require 2 new libraries? That does not make sense. bnd-baseline-maven-plugin does not depend on bnd-maven-plugin so why could/would it care that the dependencies of bnd-maven-plugin have changed?

Reply:
this is purely an issue with tests cases.

Reply:
because the invoker plugin used to "test" the maven plugins doesn't execute in the test phase, so putting the deps in test scope still breaks. but I'm investigating other options as well...

Reply:
adding the dependencies to the invoker plugin didn't work. looking for something else to try...

Reply:
The only thing that has worked so far is using scope `runtime`.

I tried the configuration `extraArtifacts` [1] of the `invoker-plugin:install` but it also fails because it doesn't find the `bnd-maven-plugin`. I'm not sure what else to do.

[1] https://maven.apache.org/plugins/maven-invoker-plugin/install-mojo.html#extraArtifacts

Reply:
Once #2480 is merged into master, rebase on master and then you can remove these dependencies.

Reply:
#2480 is now merged into master, so you can rebase on master and remove the dependencies from bnd-baseline-maven-plugin.